protocol/scripts/governance/bridge_vesters_set_rewards.py

```diff
@@ -137,8 +153,8 @@ def estimate_block_height(current_block_height, current_block_timestamp, average
 
 # Add delayed messages
 for delayed_block_number, new_fee_multiplier in [
-    (rewards_update_1_block, REWARDS_MULTIPLIER_UPDATE_1), 
-    (rewards_update_2_block, REWARDS_MULTIPLIER_UPDATE_2),
+    (delay_blocks_update_1, REWARDS_MULTIPLIER_UPDATE_1), 
```

Comment:
just to confirm, the delay msg module uses this as X where `execute @ height = current height + X ` if `X = delay_blocks_update_1`

wanted to confirm that this is not `execute @ height = X`

Reply:
I think this is correct given that the `delay_blocks` ([code](https://github.com/dydxprotocol/v4-chain/blob/88843377837e630019936ff58667dde6e6b0c0f1/proto/dydxprotocol/delaymsg/tx.proto#L26C1-L27)) is the number of blocks to delay this by. So when this msg is received by the delay block module, it will basically do:

current height + `delay_blocks` to determine when this delay msg should execute

Reply:
Yes, can confirm. Similarly, delayed_message for bridge transfer is always [86400](https://github.com/dydxprotocol/v4-chain/blob/88843377837e630019936ff58667dde6e6b0c0f1/protocol/x/bridge/keeper/acknowledge_bridges.go#L114)